classes/mail/mailables/ReviewRemindAuto.php

```diff
@@ -58,5 +59,8 @@ public function setData(?string $locale = null): void
         parent::setData($locale);
 
         $this->setOneClickAccessUrl($this->context, $this->reviewAssignment);
+
+        // See pkp/pkp-lib#9111
+        $this->addData(['lostPasswordUrl' => $this->viewData[ContextEmailVariable::PASSWORD_LOST_URL]]);
```

Comment:
Hi @Vitaliy-1, one PS client had the `passwordResetUrl` variable for this email, I was going to do a fast commit and include it, but perhaps there's something more to review.

Reply:
Can you explain I'm more details what is happening?

Reply:
This is a recent OJS 3.4 upgrade, the template `Review Reminder (Automated)` is using the variable `{$passwordResetUrl}`, but looks like it doesn't exist.

I've updated the application to use the latest commits, and the problem wasn't solved, so the email is sent with the plain-text `{$passwordResetUrl}`.